Shopify/cli
# [Fix] Do not overwrite `theme` parameter if present

### WHY are these changes introduced?

Fixes #1309

### WHAT is this pull request doing?

- Creates (or fetches from `themeConf`) development theme ONLY if `--theme` parameter is NOT given.

### How to test your changes?

1. Run `shopify theme info` to find out your development theme ID (if present)
2. Run `shopify theme list` to get a list of theme IDs you can use
3. Run `shopify theme dev --theme <Theme ID ≠ Dev Theme ID>` and make sure that the correct theme is served
4. Run `shopify theme dev` and make sure that Dev Theme ID is used (or that a new Dev Theme is created)

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
